sonic_installer/common.py

```diff
@@ -0,0 +1,27 @@
+
+import signal
```

Comment:
Add shebang line?

Reply:
Shebangs only serve a purpose in executable scripts. I prefer to keep them out of library files, as they are not meant to be executed directly. However, there is benefit in otherwise labeling a file with the Python version it was written for. Maybe simply a header docstring for library files?

Reply:
I agree with Joe regarding a the shebang. It's not necessary unless the python file is +x and meant to be executed directly.
I can add a docstring to all the files.